packages/pigeon/lib/kotlin_generator.dart

```diff
@@ -754,5 +754,5 @@ String _cast(String variable,
   if (kotlinType == 'Any?' || (safeCast && kotlinType == 'Any')) {
     return variable;
   }
-  return '$variable as${safeCast ? '?' : ''} $kotlinType';
```

Comment:
The replaceAll here seems hacky. Can we replace `String kotlinType` with one of the type classes, so that nullability is encoded separately from the base type string?

Reply:
yep, I meant to fix that this morning but it slipped my mind.